src/preproc/preprocessing_pass_context.h

```diff
@@ -0,0 +1,49 @@
+/*********************                                                        */
+/*! \file preprocessing_pass_api.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **  Justin Xu
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2017 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Preprocessing pass API for passes
+ **
+ ** The preprocessing pass API is the interface between solver and
+ ** preprocessing passes. Passes are expected to use API for 
+ ** read only access.
+ **/
+#include "cvc4_private.h"
```

Comment:
This has to be consistent with preprocessing_pass.h. If that is public, this needs to be too.